platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
+PLATFORM_DIR=/usr/share/sonic/platform
+
 rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
 supervisorctl start syncd
 
+# If this platform has an initialization file for the Broadcom LED microprocessor, load it
+if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
+    /usr/bin/bcmcmd "rcload ${PLATFORM_DIR}/led_proc_init.soc"
```

Comment:
I have not had any issues on the Dell S6000, but I have now increased the bcmcmd command timeout to 60 seconds. This should suffice for now; we can add the loading of the SOC file to SAI at a later date.